### PR TITLE
add kevin's profile to the team page

### PR DESCRIPTION
**Pull request by ChingChang9**
Preview:
image

Feel free to merge if you think it's good.

### CODE REVIEW

File: src/views/Team.vue [hunk 1]
```diff
@@ -60,12 +60,16 @@ export default {
...
-        //  title: "VP of Finance",
-        //  photo: "kevin",
-        //  story: []
-        //},
+        {
+          name: "Kevin Ang",
+          title: "VP of Finance",
+          photo: "kevin",
+          story: ["Kevin is an international student from Indonesia. " +
+            "He is 18 years old and he's going to UofA Business school majoring in finance. " +
```
Comment (marmikpatel19) [inline src/views/Team.vue:68]: "He is 18 years old and he's going to UofA Business **school*,* majoring in finance. "

File: src/views/Team.vue [hunk 2]
```diff
@@ -60,12 +60,16 @@ export default {
...
+            "He has had a passion in finance since he was in middle school, " +
```
Comment (marmikpatel19) [inline src/views/Team.vue:69]: period at the end

File: src/views/Team.vue [hunk 3]
```diff
@@ -60,12 +60,16 @@ export default {
...
+            "He has chosen to join PlantForever because he want to contribute and make an impact to the world, " +
```
Comment (marmikpatel19) [inline src/views/Team.vue:70]: "want" -> wants", "to the world" -> "in the world"
Contribute and make an impact mean the same thing; we can probably change it to just "make an impact"

File: src/views/Team.vue [hunk 4]
```diff
@@ -60,12 +60,16 @@ export default {
...
+            "mainly in the climate change issue as we’ve seen a lot of disasters caused by climate change lately."]
```
Comment (marmikpatel19) [inline src/views/Team.vue:71]: "especially with the climate crisis as lots of recent disasters have been caused by it."

**Comment by marmikpatel19** [comment]
I left some comments about grammatical changes. Feel free to accept whatever you think is right. If you want, you can send a screenshot of the final version to Kevin to make sure he's fine with it (I think he will be).

**Comment by ChingChang9** [comment]
checked in with kevin

**Pull request merged**